surfactant/cmd/cli.py

```diff
@@ -5,12 +5,26 @@
 import click
 from loguru import logger
 
+from surfactant.cmd.cli_commands import Load, Save
 from surfactant.plugin.manager import find_io_plugin, get_plugin_manager
 from surfactant.sbomtypes._relationship import Relationship
 from surfactant.sbomtypes._sbom import SBOM
 from surfactant.sbomtypes._software import Software
 
 
+@click.argument("sbom", type=click.File("r"), required=True)
+@click.option(
+    "--input_format",
+    is_flag=False,
+    default="surfactant.input_readers.cytrics_reader",
```

Comment:
We can use the ConfigManager so that if users have set a preferred default input format it will be used instead.
```suggestion
    default=ConfigManager().get(
        "core", "input_format", fallback="surfactant.input_readers.cytrics_reader"
    ),
```